### PR TITLE
Ruslan Kudaybergenov Task2

### PR DESCRIPTION
**Pull request by KRTone**
ver. 1
Check, please.

### CODE REVIEW

File: Task2 - Delegates/Ruslan Kudaybergenov/Task2/Implementations/Board.cs [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            this.boardSizeX = 10;
+            this.boardSizeY = 10;
+            this.BoardPositionX = Console.CursorLeft;
+            this.boardPositionY = Console.CursorTop;
+            this.boardLineMarkerX = "-";
+            this.boardLineMarkerY = "|";
+        }
+        public Board(int sizeX,int sizeY)
+        {
+            this.boardSizeX = sizeX;
```
Comment (BogdanVU) [inline Task2 - Delegates/Ruslan Kudaybergenov/Task2/Implementations/Board.cs]: Don't you need to set BoardPosition and boardLineMarkers in this constructor realization?)

File: Task2 - Delegates/Ruslan Kudaybergenov/Task2/Program.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+namespace Task2
+{
+    class Program
+    {
+        delegate void DrawingLines(IBoard board);
+        static DrawingLines drawingLines = null;
+        static void Main(string[] args)
+        {
+            var board = new Board();
```
Comment (BogdanVU) [inline Task2 - Delegates/Ruslan Kudaybergenov/Task2/Program.cs]: You use classes, that already implement intefaces, but not the interfaces.

Comment (KRTone) [reply Task2 - Delegates/Ruslan Kudaybergenov/Task2/Program.cs]: repeat, plz, what do u mean

Comment (KRTone) [reply Task2 - Delegates/Ruslan Kudaybergenov/Task2/Program.cs]: repeat, plz, what do u mean? 
If i understood correctly u said about `using Task2.Abstractions;` and i needed it to create this `delegate void DrawingLines(**IBoard** board);`

File: Task2 - Delegates/Ruslan Kudaybergenov/Task2/Implementations/Board.cs (hunk 1 above)
Comment (KRTone) [reply Task2 - Delegates/Ruslan Kudaybergenov/Task2/Implementations/Board.cs]: :D yeah, it's mistake(slak-baked) and trash that needed to delete. I used just default constructor

File: Task2 - Delegates/Ruslan Kudaybergenov/Task2/Program.cs (hunk 2 above)
Comment (BogdanVU) [reply Task2 - Delegates/Ruslan Kudaybergenov/Task2/Program.cs]: As i understood, you use IBoard to define DrawLines delegate. But for what did you develop other interfaces?

Comment (KRTone) [reply Task2 - Delegates/Ruslan Kudaybergenov/Task2/Program.cs]: I understood. But i think that drawingline(IBoard board) will be more usefull then drawingline(Board board) coz i can use every one implementation of IBoard interface(for example Triangle:IBoard, Circle:IBoard). Or is it not good idea?

Comment (BogdanVU) [reply Task2 - Delegates/Ruslan Kudaybergenov/Task2/Program.cs]: There is no issues in your IBoard using. My comment is about using other declared intefaces.

Comment (KRTone) [reply Task2 - Delegates/Ruslan Kudaybergenov/Task2/Program.cs]: Emm, are u talking about `drawingline(Abstractions.IBoard board);` and usless `using Task2.Abstractions;` in class with 'Main' method?
or about inherited interfaces by classes?

**Comment by chesdenis** [comment]
@Chesanita or @BogdanVU  can you please review/approve?

**Pull request merged**